test/e2e/packetcapture_test.go

```diff
@@ -807,8 +952,19 @@ func runPacketCaptureTest(t *testing.T, data *TestData, tc pcTestCase) {
 	if tc.expectedStatus.NumberCaptured == 0 {
 		return
 	}
+
+	var targetPodRef *crdv1alpha1.PodReference
+	if tc.pc.Spec.Source.Pod == nil {
+		targetPodRef = tc.pc.Spec.Destination.Pod
+	} else {
+		targetPodRef = tc.pc.Spec.Source.Pod
+	}
```

Comment:
here again, a comment is needed
I am assuming that by "target" Pod, you mean the Pod where the packet capture will run, and that you are just mirroring the behavior of the PacketCaptuire controller, which defaults to the source Pod if available